python/pyspark/streaming/tests.py

```diff
@@ -1297,4 +1299,5 @@ def search_kinesis_asl_assembly_jar():
     for testcase in testcases:
         sys.stderr.write("[Running %s]\n" % (testcase))
         tests = unittest.TestLoader().loadTestsFromTestCase(testcase)
-        unittest.TextTestRunner(verbosity=3).run(tests)
+        unittest.main(tests, verbosity=3,
+            testRunner=xmlrunner.XMLTestRunner(output='target/surefire-reports'))
```

Comment:
this should be aligned with `tests,`. you can run `dev/lint-python` to check style locally.